mocks/firestore.js

```diff
@@ -30,7 +30,7 @@ const _randomId = () => Math.floor(Math.random() * Number.MAX_SAFE_INTEGER).toSt
 
 class FakeFirestore {
   constructor(stubbedDatabase = {}, options = {}) {
-    this.database = stubbedDatabase;
+    this.database = timestamp.convertTimestamps(stubbedDatabase);
```

Comment:
This is only necessary on creation.  Any subsequent edits can be assumed to contain timestamp where appropriate already